pkg/stack/function_helpers.go

```diff
@@ -22,7 +22,7 @@ import (
 	"path/filepath"
 )
 
-const DefaulMembraneVersion = "v0.12.1-rc.5"
+const DefaultMembraneVersion = "v0.13.0-rc.12"
```

Comment:
our "make generate" should automatically update this from our go.sum :thinking:

Reply:
Good idea, I've been doing a bit more reading on doing this with go generate. Would the idea in this case be that we could create a seperate gen script for individual packages with a generate directive to create that?

In this specific case would we generate a new file e.g. pkg/stack/const.go with this content but have the generate directive in the source file that depends on this constant?

Reply:
https://github.com/nitrictech/newcli/pull/48